doc/modules/feature_selection.rst

```diff
@@ -261,6 +259,50 @@ meta-transformer)::
     * :ref:`sphx_glr_auto_examples_ensemble_plot_forest_importances_faces.py`: example
       on face recognition data.
 
+.. _sequential_feature_selection:
+
+Sequential Feature Selection
+============================
+
+Sequential Feature Selection [sfs]_ (SFS) is available in the
+:class:`~sklearn.feature_selection.SequentialFeatureSelector` transformer.
+SFS can be either forward and backward:
+
+Forward-SFS is a greedy procedure that iteratively finds the best new feature
+to add to the set of selected features. Concretely, we initially start with
+zero feature and find the one feature that maximizes a cross-validated score
+when an estimator is trained on this single feature. Once that first feature
+is selected, we repeat the procedure by adding a new feature to the set of
+selected features. The procedure stops when the desired number of selected
+features is reached, as determined by the `n_features_to_select` parameter.
+
+Backward-SFS follows the same idea but works in the opposite direction:
+instead of starting with no feature and greedily adding features, we start
+with *all* the features and greedily *remove* features from the set. The
+`forward` parameter controls whether forward or backward SFS is used.
+
+In general, forward and backward selection do not yield equivalent results.
+Also, one may be much faster than the other depending on the requested number
+of selected features: if we have 10 features and ask for 7 selected features,
+forward selection would need to perform 7 iterations while backward selection
+would only need to perform 3.
+
+SFS differs from :class:`~sklearn.feature_selection.RFE` and
+:class:`~sklearn.feature_selection.SelectFromModel` in that it does not
+require the underlying model to expose a `coef_` or `feature_importances_`
+attribute. It may however be slower considering that more models need to be
+evaluated, compared to the other approaches.
```

Comment:
Maybe be even more specific: "For example in forward selection, the iteration going from ``k`` features to ``k-1`` features using 5-fold cross-validation requires fitting ``5 * k`` many models, while ``RFE`` would require only a single fit, and ``SelectFromModel`` always just does a single fit and requires no iterations." or something like that?

Reply:
Or making the "more specific" working more "general" ;)

"For example in forward selection, the iteration going from m features to m-1 features using k-fold cross-validation requires fitting k * m many models, while RFE would require only a single fit, and SelectFromModel always just does a single fit and requires no iterations."

Reply:
done